src/multicellular_stage/CellBodyPlanInternalCalculations.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Collections.Generic;
+using Godot;
+
+public static class CellBodyPlanInternalCalculations
+{
+    public static Dictionary<Compound, float> GetTotalSpecificCapacity(IEnumerable<CellTemplate> cells,
+        out float nominalCapacity)
+    {
+        nominalCapacity = 0.0f;
+
+        var capacities = new Dictionary<Compound, float>();
+
+        foreach (var cell in cells)
+        {
+            var totalNominalCap = MicrobeInternalCalculations.GetTotalNominalCapacity(cell.Organelles);
+            nominalCapacity += totalNominalCap;
+
+            MicrobeInternalCalculations.AddSpecificCapacity(cell.Organelles, capacities, totalNominalCap);
+        }
+
+        return capacities;
+    }
+
+    public static float CalculateSpeed(IReadOnlyList<HexWithData<CellTemplate>> cells)
+    {
+        var leader = cells[0].Data!;
+
+        var speed = MicrobeInternalCalculations.CalculateSpeed(leader.Organelles, leader.MembraneType,
+            leader.MembraneRigidity, leader.IsBacteria);
+
+        speed *= cells.Count * Constants.CELL_COLONY_MOVEMENT_FORCE_MULTIPLIER;
+        var seriesValue = 1 - 1 / (float)Math.Pow(2, cells.Count - 1);
+        speed -= speed * 0.15f * seriesValue;
+
+        if (cells.Count == 1)
+            return speed;
```

Comment:
Shouldn't this return be before the speed is adjusted? Because if a single cell I don't think multicellular stage creates the colony, so it should have the exact same speed as in the microbe stage.